pkg/kubelet/kubelet.go

```diff
@@ -1370,7 +1370,7 @@ func (kl *Kubelet) initializeRuntimeDependentModules() {
 	kl.StatsProvider.GetCgroupStats("/", true)
 	// Start container manager.
 	node, err := kl.getNodeAnyWay()
-	if err != nil {
+	if err != nil && err.Error() != ErrNodeInfoCacheNotReady.Error() {
```

Comment:
this sort of string typing of errors is fragile and is not something we do in code we control (sometimes we have no choice when recognizing errors from the stdlib, etc)